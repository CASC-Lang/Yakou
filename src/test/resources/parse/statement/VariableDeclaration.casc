class VariableDeclaration

impl VariableDeclaration {
    comp {
        a := 1
        mut b := 1
        c, d := 1
        mut e, f := 1
        g, mut h := 1
        mut i, mut j := 1
        k, m, l, m := 1, 2, 3, 4
        mut
    }
}