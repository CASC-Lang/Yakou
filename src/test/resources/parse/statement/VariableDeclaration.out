error: Unexpected `mut` keyword
--> ./VariableDeclaration.casc:12:8
11 |         k, m, l, m := 1, 2, 3, 4
12 |         mut
             ^^^= hint: Do you mean to declare variable?
13 |     }